### ISSUE TITLE
Standard link color too close to text color

### ISSUE THREAD

**Issue by dajoturner** [ucsc/wcms-design-templates#344]
Jane Nyberg reported this issue. She's saying that the blue of a link is too close to the standard text black, so that it's not obvious to her eyes that something is a link. I don't have that problem, but I can see what she's saying. I let her know that this is already an approved design change, but I'm opening this with an accessibility tag. We can evaluate.

Jane said:
I'm sure you've already heard about this, but I might as well give you my feedback. It is very hard to tell the difference between a link and normal text in WCMS. I was about to send in a correction for a security page saying that it needed a link, but after my my mouse rolled over some of the text I saw there already was a link.
